uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/b2c/GoogleLoginComponentHandler.java

```diff
@@ -85,4 +88,18 @@ public void handleNextButton() {
             throw new AssertionError(e);
         }
     }
+
+    public void handleRecoveryEmail() {
+        Logger.i(TAG, "Handle Google Recovery Email UI..");
+        final UiObject confirmationEmailButton = UiAutomatorUtils.obtainUiObjectWithText(this.RECOVERY_EMAIL_BUTTON_TEXT);
+        if (confirmationEmailButton.exists()){
+            try {
+                confirmationEmailButton.click();
+                UiAutomatorUtils.handleInput(this.RECOVERY_EMAIL_INPUT_RESOURCE_ID, this.RECOVERY_EMAIL);
+                handleNextButton();
+            } catch (final UiObjectNotFoundException e) {
+                throw new AssertionError(e);
+            }
+        }
```

Comment:
I'm personally not a fan of doing these UI element checks with `if` blocks...because these just mean adding to the test execution time....I know we talked about this earlier, but can you remind why emailConfirmationButton may show up versus not?

Reply:
from what I know the emailConfirmationButton  will only appear the first time you login in a new device, I am not so convinced myself to have this handler, if we remove it we may have to handle the first run manually, thoughts?

Reply:
Let's also wrap this in a boolean flag...and set that to `false` to begin with and test whether that works on Firebase or not.

Reply:
Apply change here d21cd2bbfe809f425adf0d4bf186efe8bd91cdad
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/d21cd2bbfe809f425adf0d4bf186efe8bd91cdad/uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/b2c/GoogleLoginComponentHandler.java#L94